components/gsma-authenticators/src/main/java/com/wso2telco/gsma/authenticators/voice/ValidSoftJsonHelper.java

```diff
@@ -0,0 +1,88 @@
+package com.wso2telco.gsma.authenticators.voice;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.codehaus.jettison.json.JSONException;
+import org.codehaus.jettison.json.JSONObject;
+
+/**
+ * Created by sheshan on 5/4/17.
+ */
+
+public class ValidSoftJsonHelper {
+
+    private String isUserActiveRequestJson;
+    private String userRegistrationAndAuthenticationJson;
+    private final String UNKNOWN_USER = "UNKNOWN_USER";
+    private final String ACTIVE = "ACTIVE";
+
+    private static Log log = LogFactory.getLog(ValidSoftJsonHelper.class);
+
+    public String getIsUserActiveRequestJson() {
+        return isUserActiveRequestJson;
+    }
+
+    public void setIsUserActiveRequestJson(String logId , String msisdn ) {
+
+        // remove msisdn modifiation line this from when going to prod
+               msisdn = "0094"+ msisdn.substring(3,12);
+        //
+        String jsonTemplate = "{" +
+                "\"serviceData\": {" +
+                "\"serviceId\": \"PasswordResetBiometric\"," +
+                "\"loggingId\": \""+logId+"\"" +
+                "}," +
+                "\"userData\": {" +
+                "\"identifier\": \""+msisdn+"\"" +
+                "}" +
+                "}";
+        log.info("isUserActiveRequest Json ::::: " + jsonTemplate);
+        this.isUserActiveRequestJson = jsonTemplate;
+    }
+
+    public void setUserRegistrationAndAuthenticationJson(String logId , String msisdn){
+
+        // remove msisdn modifiation line this from when going to prod
+        msisdn = "94"+ msisdn.substring(3,12);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "msisdn". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS1226)